packages/cli/src/html/sequenceDiagram.js

```diff
@@ -16,7 +16,7 @@ async function initializeApp() {
   return new Vue({
     el: '#app',
     render: (h) =>
-      h(VDiagramSequence, {
```

Comment:
I'd split this into a separate commit due to the fact this is a separate package.

Reply:
:heavy_check_mark: